public_html/wp-content/mu-plugins/blocks/source/blocks/organizers/block-controls.js

```diff
@@ -61,28 +61,34 @@ export class BlockControls extends Component {
 			default :
 				output = (
 					<Placeholder
-						className={ classnames( 'wordcamp-block-edit-placeholder', 'wordcamp-block-edit-placeholder-no-mode' ) }
+						className={ classnames( 'wordcamp__edit-placeholder', 'has-no-mode' ) }
```

Comment:
`MUST` - I think this should be `wordcamp-organizers__block-contents wordcamp-placeholder` for similar reasons as earlier comments.

> Notably, if your folder contains multiple files, each with their own default exported component, only the element rendered by that of index.js can be considered the root. All others should be treated as descendents.

There's probably other places that need a similar change.

Reply:
So `wordcamp__edit-placeholder` actually applies a shared style defined in `z-index.scss`, so it would make sense to keep it as it is. What do you think?

Reply:
🤷‍♂ , that works for me.